apis/urls.py

```diff
@@ -1,70 +1,78 @@
 from django.conf import settings
-from django.conf.urls import url, include
+from django.urls import include, re_path
 from django.contrib import admin
 from django.urls import path
 
 from apis_core.apis_entities.api_views import GetEntityGeneric
 
 if "theme" in settings.INSTALLED_APPS:
     urlpatterns = [
-        url(r"^apis/", include("apis_core.urls", namespace="apis")),
-        url(r"^api-auth/", include("rest_framework.urls", namespace="rest_framework")),
+        re_path(r"^apis/", include("apis_core.urls", namespace="apis")),
```

Comment:
we should use `re_path` only when regex is really needed. for normal routes `path` is enough. eg `path("apis/", include("apis_core.urls", namespace="apis")),`